README.md

```diff
@@ -157,6 +162,7 @@ Table of Contents
   * [Abstract API](https://www.abstractapi.com) — API suite for a variety of use cases including IP geolocation, gender detection or even email validation.
   * [algorithmia.com](https://algorithmia.com/) — Host algorithms for free. Includes free monthly allowance for running algorithms. Now with CLI support.
   * [Apify](https://www.apify.com/) — Web scraping and automation platform that lets you create an API extracting websites data. Free tier with 10k monthly crawls and 7 days data retention.
+  * [Azure Cognitive Services](https://azure.microsoft.com/services/cognitive-services/?WT.mc_id=docs-freefordev-yolasors) - AI/ML APIs (Computer Vision, Translator, Face detection, Bots...) with free tier including limited transactions
```

Comment:
please do not duplicate, put all the azure stuff in one place at the top in the big list

Reply:
Sorry I thought your comment was only for this section, as there was other duplicates prior to my commit. I'll also remove those.